client/src/utils/components/markdown/RenkuMarkdownWithPathTranslation.js

```diff
@@ -223,7 +223,7 @@ function RenkuMarkdownWithPathTranslation(props) {
   }
 
   const previewLinks = divWithMarkdown.getElementsByTagName("a");
-  const fullBaseUrl = props.client.baseUrl.replace("/api", "/projects")
+  const fullBaseUrl = props.client.baseUrl.replace("/ui-server/api", "/projects")
```

Comment:
```suggestion
  const fullBaseUrl = props.client.uiserverUrl.replace("/ui-server", "/projects")
```
Not a big change tbf since "baseUrl" is manually defined as "uiserverUrl" + "/api" :grin: 

Actually, the best solution would be using the `params["BASE_URL"]` variable that contains the pure UI URL. That would be robust to deploying UI and ui-server to different URLs. It sounds confusing, but `client.baseUrl` uses `params["UISERVER_URL"]`

Reply:
Yes, I didn't like that bit of code much either. I switched to using `Url.get(Url.pages.projects)`, which is, I think, an even better solution.